test/format/t.c

```diff
@@ -218,8 +218,12 @@ main(int argc, char *argv[])
      * file, used when running checks.
      */
     if (g.reopen) {
-        if (config != NULL)
-            testutil_die(EINVAL, "-c incompatible with -R");
+        /*
+         * The test-format binary can now reopen an already existing database with different CONFIG
+         * file because some configurations are not supported by the older release and in
+         * compatibility testing we want the test format binary to operate on the CONFIG that it
+         * understands.
+         */
```

Comment:
Thanks for adding the descriptive comment. Can a blank line be added after this new comment section, so that it does not look to be related to the line (227) immediately after it?

Reply:
I'm not sure if this comment is a good idea.

What I'm trying to say is code doesn't generally have comments describing  "things we don't do", that's going to get messy if we keep adding comments describing things the code doesn't do.

Reply:
@keithbostic Yeah, I agree not to mention the things that we don't do. I will better add this comment in the PR descriptions.